### DISCUSSION TITLE
Maintaining Burning-Series or go Anime-Exclusive?

### DISCUSSION THREAD

**Discussion by DatL4g** [DATL4G/BurningSeries-Android#57] [Polls]
I started this project 2 years ago and since then the current tech-stack changed a lot, the best example is that this app got a revamp compatible with PC.
Now the revamp already is hard to maintain as some things where experimental when developed and changed as they got stable, these effect e.g. visuals.

**I probably can't avoid a complete rebuild either way.**

## Personal view

I know that Burning-Series is much more than Anime, however I hardly watch anything other and in my opinion it's the most "painful" part of the app, as these get deleted more often.

Additionally it has always been the most important and worked on part of the app.

## Financial view

This project costs me mothly money.
I'm happy for the support of @ngelus and other one-time supporter, however it rarely evens out.
A new app with anime only focus would reduce the costs.

## Development view

As mentioned earlier, this project needs a rebuild sooner or later anyway.
A complete new app could mean new features, support for more websites, and maybe even Browser support.
Browser support means no Android or Desktop app required, but of course still the main point.

# Your opinion

I want your thinkings about this topic, as dropping Burning-Series means dropping "casual" series but improving Anime a lot.
A poll is appended so I can see your reaction, feel free to comment but keep it topic related.

### My decision

The poll influences my decision however keep in mind that this is my project and even if it's more "keep Burning-Series" leaning, I might decide overwise.

**Comment by High-village** [comment]
ngl I really don't care about anime too much, there are so many apps for specifically anime that I can already use and do sometimes. I use this app for BS on purpose for german content, german series, german documentaries but def not for watching anime really tbh..
it would be a shame to not have an app again for that kind of german content with atm still the biggest streaming database for it, specially since I just found this app 2 weeks ago..

**Comment by ngelus** [comment]
I've always been watching on Burning-Series and have never heard of "Go Anime", so I will have to look into that first, but I also don't just use the app for anime. So I'd honestly rather keep it Burning-Series, but I won't be mad if you want to change it to "Go Anime" either way. That would just mean that I'd have to go to the Burning-Series website again for my other series. This is your project and if you feel like this is the route you want to go then you should really just take it. Imma support you either way you decide.

**Reply by DatL4g** [reply]
I think there's nothing like "Go Anime", I meant "go Anime-Exclusive" instead. :)

**Reply by ngelus** [reply]
> I think there's nothing like "Go Anime", I meant "go Anime-Exclusive" instead. :)

Well... As I've said before; I don't use this exclusively for anime, so it would be kinda sad to see the feature for watching other series go, but I will still be supporting this if you decide to make it anime only.
@DATL4G

**Comment by ghost** [comment]
I would recommend to go anime exclusive in order to reduce costs and keep the project alive.
As a second step I would recommend setting up a budget goal, so each supporter knows how much they need to donate.
Then, when the anime exclusive version is viable, you can put more resources into expanding the project.
Don't start too big in the beginning

**Comment by ghost** [comment]
Have you considered adopting other similar projects? For example [https://github.com/stantanasi/streamflix](https://github.com/stantanasi/streamflix) features a similar UI. Porting the existing html-parser should be far less effort than before.
In this manner you could add both anime and bs as a provider.

**Comment by operationairstrike** [comment]
But why would you make the app less useful? If its a resource problem just make the client do more of the loading. And otherwise make the user choose if they want everything or anime only at the first start of the app.
